### ISSUE TITLE
swaync notifications appear on top of lockscreen

### ISSUE THREAD

**Issue by lcolitti** [WayfireWM/wayfire#2105]
**Describe the bug**
When running swaync, notifications appear above the lockscreen

This is because swaync uses "overlay" as the default layer for notifications, but wayfire doesn't seem to have separate overlay and lock layers. (There used to be a LAYER_LOCK, but in d27894a7 that got turned into to OVERLAY.) So the lockscreen created by ext_session_lock is on the overlay layer, same as the notifications.

This seems wrong. I think overlay is supposed to be for desktop widgets but the lock screen should be above these.

Should there be a LOCKSCREEN layer that's above OVERLAY?

**To Reproduce**
1. Install swaylock and swaync
2. swaync &
3. swaylock & (sleep 1; notify-send hello)
4. See the notifications appear on top of the lockscreen.

**Expected behavior**
The notifications should be hidden by the lockscreen.

**Wayfire version**
git

**Comment by lcolitti** [comment]
I spent hours looking at the swaylock code trying to figure out how this works and I finally realized that it probably **doesn't** work. swaylock deleted the layer-shell support [this commit](https://github.com/swaywm/swaylock/commit/ac3b49b6571ceda3f8db11a98bfe320106996280), from which point it only supports the ext_session_lock protocol, which as far as I can tell wayfire doesn't support. It just happens to work on my machine because my copy of swaylock is old.

Are/were there any plans to implement support for ext-session-lock-v1? That seems like the only/right way to fix this issue.
